ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host2/aaa-root.yml

```diff
@@ -0,0 +1,3 @@
+---
+## AAA Root for default config
+generate_default_config: true
```

Comment:
Do we need to generate default config here in regards to aaa/radius/tacacs?

Reply:
Yes in regards to aaa for improve the coverage.

Reply:
Please use the aaa_root knob instead of generate_default_config which is deprecated.

Reply:
Thanks.
Removed the file from host2 as having converge issue for not defining the generate_default_config. As decided deprecated keys will handle separately.